? art/category_weather_cloudy_16.png
? art/category_weather_fog_16.png
? art/category_weather_partly_cloudy_16.png
? art/category_weather_rain_16.png
? art/category_weather_snow_16.png
? art/category_weather_sun_16.png
? art/category_weather_tstorm_16.png
Index: art/Makefile.am
===================================================================
RCS file: /cvs/gnome/gal/art/Makefile.am,v
retrieving revision 1.4
diff -u -r1.4 Makefile.am
--- art/Makefile.am	8 Jan 2003 19:16:12 -0000	1.4
+++ art/Makefile.am	30 Dec 2004 03:08:28 -0000
@@ -5,8 +5,8 @@
 	category_favorites_16.png		\
 	category_gifts_16.png			\
 	category_goals_16.png			\
-	category_holiday_16.png			\
 	category_holiday-cards_16.png		\
+	category_holiday_16.png			\
 	category_hot-contacts_16.png		\
 	category_ideas_16.png			\
 	category_international_16.png		\
@@ -17,7 +17,14 @@
 	category_status_16.png			\
 	category_strategies_16.png		\
 	category_suppliers_16.png		\
-	category_time-and-expenses_16.png
+	category_time-and-expenses_16.png	\
+	category_weather_cloudy_16.png		\
+	category_weather_fog_16.png		\
+	category_weather_partly_cloudy_16.png	\
+	category_weather_rain_16.png		\
+	category_weather_snow_16.png		\
+	category_weather_sun_16.png		\
+	category_weather_tstorm_16.png
 
 EXTRA_DIST = \
 	$(images_DATA)
Index: gal/widgets/e-categories-master-list-array.c
===================================================================
RCS file: /cvs/gnome/gal/gal/widgets/e-categories-master-list-array.c,v
retrieving revision 1.14
diff -u -r1.14 e-categories-master-list-array.c
--- gal/widgets/e-categories-master-list-array.c	16 Nov 2002 23:42:17 -0000	1.14
+++ gal/widgets/e-categories-master-list-array.c	30 Dec 2004 03:08:29 -0000
@@ -64,27 +64,34 @@
 	gchar *category;
 	gchar *icon;
 } builtin_categories[] = {
-	{ N_("Birthday"),         GAL_IMAGESDIR "/category_birthday_16.png" },
-	{ N_("Business"),         GAL_IMAGESDIR "/category_business_16.png" },
-	{ N_("Competition"),      NULL },
-	{ N_("Favorites"),        GAL_IMAGESDIR "/category_favorites_16.png" },
-	{ N_("Gifts"),            GAL_IMAGESDIR "/category_gifts_16.png" },
-	{ N_("Goals/Objectives"), GAL_IMAGESDIR "/category_goals_16.png" },
-	{ N_("Holiday"),          GAL_IMAGESDIR "/category_holiday_16.png" },
-	{ N_("Holiday Cards"),    GAL_IMAGESDIR "/category_holiday-cards_16.png" },
-	{ N_("Hot Contacts"),     GAL_IMAGESDIR "/category_hot-contacts_16.png" },
-	{ N_("Ideas"),            GAL_IMAGESDIR "/category_ideas_16.png" },
-	{ N_("International"),    GAL_IMAGESDIR "/category_international_16.png" },
-	{ N_("Key Customer"),     GAL_IMAGESDIR "/category_key-customer_16.png" },
-	{ N_("Miscellaneous"),    GAL_IMAGESDIR "/category_miscellaneous_16.png" },
-	{ N_("Personal"),         GAL_IMAGESDIR "/category_personal_16.png" },
-	{ N_("Phone Calls"),      GAL_IMAGESDIR "/category_phonecalls_16.png" },
-	{ N_("Status"),           GAL_IMAGESDIR "/category_status_16.png" },
-	{ N_("Strategies"),       GAL_IMAGESDIR "/category_strategies_16.png" },
-	{ N_("Suppliers"),        GAL_IMAGESDIR "/category_suppliers_16.png" },
-	{ N_("Time & Expenses"),  GAL_IMAGESDIR "/category_time-and-expenses_16.png" },
-	{ N_("VIP"),              NULL },
-	{ N_("Waiting"),          NULL },
+	{ N_("Birthday"),               GAL_IMAGESDIR "/category_birthday_16.png" },
+	{ N_("Business"),               GAL_IMAGESDIR "/category_business_16.png" },
+	{ N_("Competition"),            NULL },
+	{ N_("Favorites"),              GAL_IMAGESDIR "/category_favorites_16.png" },
+	{ N_("Gifts"),                  GAL_IMAGESDIR "/category_gifts_16.png" },
+	{ N_("Goals/Objectives"),       GAL_IMAGESDIR "/category_goals_16.png" },
+	{ N_("Holiday"),                GAL_IMAGESDIR "/category_holiday_16.png" },
+	{ N_("Holiday Cards"),          GAL_IMAGESDIR "/category_holiday-cards_16.png" },
+	{ N_("Hot Contacts"),           GAL_IMAGESDIR "/category_hot-contacts_16.png" },
+	{ N_("Ideas"),                  GAL_IMAGESDIR "/category_ideas_16.png" },
+	{ N_("International"),          GAL_IMAGESDIR "/category_international_16.png" },
+	{ N_("Key Customer"),           GAL_IMAGESDIR "/category_key-customer_16.png" },
+	{ N_("Miscellaneous"),          GAL_IMAGESDIR "/category_miscellaneous_16.png" },
+	{ N_("Personal"),               GAL_IMAGESDIR "/category_personal_16.png" },
+	{ N_("Phone Calls"),            GAL_IMAGESDIR "/category_phonecalls_16.png" },
+	{ N_("Status"),                 GAL_IMAGESDIR "/category_status_16.png" },
+	{ N_("Strategies"),             GAL_IMAGESDIR "/category_strategies_16.png" },
+	{ N_("Suppliers"),              GAL_IMAGESDIR "/category_suppliers_16.png" },
+	{ N_("Time & Expenses"),        GAL_IMAGESDIR "/category_time-and-expenses_16.png" },
+	{ N_("VIP"),                    NULL },
+	{ N_("Waiting"),                NULL },
+	{ N_("Weather: Cloudy"),        GAL_IMAGESDIR "/category_weather_cloudy_16.png" },
+	{ N_("Weather: Fog"),           GAL_IMAGESDIR "/category_weather_fog_16.png" },
+	{ N_("Weather: Partly Cloudy"), GAL_IMAGESDIR "/category_weather_partly_cloudy_16.png" },
+	{ N_("Weather: Rain"),          GAL_IMAGESDIR "/category_weather_rain_16.png" },
+	{ N_("Weather: Snow"),          GAL_IMAGESDIR "/category_weather_snow_16.png" },
+	{ N_("Weather: Sunny"),         GAL_IMAGESDIR "/category_weather_sun_16.png" },
+	{ N_("Weather: Thunderstorms"), GAL_IMAGESDIR "/category_weather_tstorm_16.png" },
 };
 
 #define BUILTIN_CATEGORY_COUNT (sizeof(builtin_categories) / sizeof(builtin_categories[0]))
